docs/SOPs/Managed_access/Data_Transfer_SOP.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Data Transfer SOP
+parent: Managed access
+grand_parent: SOPs
+nav_order: 4
+last_modified_date: 30/05/2024
+---
+
+<script src="https://kit.fontawesome.com/fc66878563.js" crossorigin="anonymous"></script>
+
+# Managed access dataset - Data Transfer SOP
+Once the data contributor has notified the HCA data wrangler that they have completed the  metadata spreadsheet, the wrangler can give them access to hca-util area
+
+## Data and metadata upload prerequisites
+To upload data to hca-util contributors need credentials and their project’s area uuid. Credentials for hca-util must be zipped and password protected. 
```

Comment:
To upload data to the AWS secure storage area using the hca-util tool, contributors need credentials and the name of the secure storage area for their project. Credentials for the hca-util tool must be zipped and password protected when sent out.

Reply:
they need the area's uuid, the name is not relevant

Reply:
I know the Name is actually the UUID, but to an external person UUID is a strange concept. They need the name of that folder (which we have called after the UUID, I understand)